### ISSUE TITLE
I encountered the following problems, how to solve?

### ISSUE THREAD

**Issue by dcl-lily** [pbanaszkiewicz/python-rrdtool#6]
PS C:\Users\Alex> pip install  rrdtool-python
Collecting rrdtool-python
  Could not find a version that satisfies the requirement rrdtool-python (from versions: )
No matching distribution found for rrdtool-python
PS C:\Users\Alex>
PS C:\Users\Alex>
PS C:\Users\Alex> pip install python-rrdtool
Collecting python-rrdtool
  Using cached python-rrdtool-1.4.7.tar.gz
Building wheels for collected packages: python-rrdtool
  Running setup.py bdist_wheel for python-rrdtool ... error
  Complete output from command "d:\program files (x86)\anaconda2\python.exe" -u -c "import setuptools, tokenize;__file__='c:\\users\\alex\\appdata\\local\\temp\\pip-build-bhnwvi\\python-rrdtool\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d c:\users\alex\appdata\local\temp\tmp2mgygapip-wheel- --python-tag cp27:
  running bdist_wheel
  running build
  running build_ext
  running build_configure
  error: [Error 193] %1 不是有效的 Win32

  ----------------------------------------
  Failed building wheel for python-rrdtool
  Running setup.py clean for python-rrdtool
Failed to build python-rrdtool
Installing collected packages: python-rrdtool
  Running setup.py install for python-rrdtool ... error
    Complete output from command "d:\program files (x86)\anaconda2\python.exe" -u -c "import setuptools, tokenize;__file__='c:\\users\\alex\\appdata\\local\\temp\\pip-build-bhnwvi\\python-rrdtool\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record c:\users\alex\appdata\local\temp\pip-3qp0uh-record\install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_ext
    running build_configure
    error: [Error 193] %1 不是有效的 Win32

    ----------------------------------------
Command ""d:\program files (x86)\anaconda2\python.exe" -u -c "import setuptools, tokenize;__file__='c:\\users\\alex\\appdata\\local\\temp\\pip-build-bhnwvi\\python-rrdtool\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record c:\users\alex\appdata\local\temp\pip-3qp0uh-record\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in c:\users\alex\appdata\local\temp\pip-build-bhnwvi\python-rrdtool\
PS C:\Users\Alex>

**Comment by diphuaji** [comment]
obviously, there is not any info that tells us the author tested the installation process on Windows.
Switch to linux if you can.
